templates/admin/action-buttons.php

```diff
@@ -2,27 +2,24 @@
 /** @var WC_Gateway_Swedbank_Pay_Cc $gateway */
 /** @var WC_Order $order */
 
-if ( ! defined( 'ABSPATH' ) ) {
-	exit;
-} // Exit if accessed directly
+defined( 'ABSPATH' ) || exit;
 
 ?>
 
-<?php if ( $gateway->can_capture( $order ) ): ?>
+<?php if ( $gateway->core->canCapture( $order->get_id() ) ) : ?>
 	<button id="swedbank_pay_capture"
 			type="button" class="button button-primary"
 			data-nonce="<?php echo wp_create_nonce( 'swedbank_pay' ); ?>"
 			data-order-id="<?php echo esc_html( $order->get_id() ); ?>">
-		<?php _e( 'Capture Payment', WC_Swedbank_Pay::TEXT_DOMAIN ) ?>
+		<?php _e( 'Capture Payment', 'swedbank-pay-woocommerce-payments' ); ?>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (like esc_html_e() or esc_attr_e()), found '_e'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)